### PR TITLE
[TDSK-237] Add `phone_number` field in `Participant`

### PR DESCRIPTION
**Pull request by mrashed-dev**
# Description
Adds support for setting and getting participant phone numbers.

# License
I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: src/main/java/com/nylas/Participant.java [hunk 1]
```diff
@@ -55,12 +65,14 @@ public Participant comment(String comment) {
...
 		Maps.putIfNotNull(params, "comment", comment);
 		return params;
 	}
 
 	@Override
 	public String toString() {
-		return "Participant [name=" + name + ", email=" + email + ", status=" + status + ", comment=" + comment + "]";
+		return "Participant [name=" + name + ", email=" + email + ", phone_number=" + phone_number
+				+ ", status=" + status + ", comment=" + comment + "]";
 	}
```
Comment (jseller) [inline src/main/java/com/nylas/Participant.java:77]: might want to use StringBuilder and append() instead of concat; its faster and more readable

Comment (jseller) [reply src/main/java/com/nylas/Participant.java:77]: any tests to run for this? Just a small one to exercise the code building these objs and getting the string representations

Comment (mrashed-dev) [reply src/main/java/com/nylas/Participant.java:82]: @jseller thanks for your feedback! I think changing this short of a concat to a StringBuilder won't offer much noticeable increase and I tried it out in terms of readability, it just gets too long for what it does. However, I have re-formatted the toString concats to be more readable, let me know if that works better! Also unfortunately we do not have any unit tests atm, but you can locally edit EventsExample.java and log a participant's toString value if that works!

File: src/main/java/com/nylas/Participant.java [hunk 2]
```diff
@@ -40,6 +45,11 @@ public Participant name(String name) {
 		return this;
 	}
 
+	public Participant phoneNumber(String phoneNumber) {
+		this.phone_number = phoneNumber;
+		return this;
+	}
```
Comment (MrFaceRoxw3ll) [inline src/main/java/com/nylas/Participant.java:51]: changelog.md

**Review by jseller — APPROVED** [review]
LG!

**Pull request merged**